templates/model/model.stub

```diff
@@ -8,6 +8,7 @@ $DOCS$
 class $MODEL_NAME$ extends Model
 {
 $SOFT_DELETE$
+    public $connection = '$CONNECTION_NAME$';
```

Comment:
somehow, can we make it optional? As lots of people will not need it.

Reply:
Sure, i'll update this soon.